docs/integrations/google/workspace/collect-logs.md

```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
 **Google Workspace Alert Center** alerts are in JSON format. Most of the alerts have a few common fields. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information, see this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
 
 
-## Configure Collection for Google Workspace Audit Source
```

Comment:
Actually we need to keep "Audit" as we are using the "G Suite Apps Audit" source, not the "Google Workspace" source. The latter only collects user data and it's for CSE mostly (I know it's confusing).